### DISCUSSION TITLE
Need to get rid of left border that shows up on tab hover

### DISCUSSION THREAD

**Discussion by manyfew** [piroor/treestyletab#3125] [Q&A]
I'm trying to get rid of a vertical border that comes up on tab hover, any ideas?

![Tree Style Tabs gray line](https://user-images.githubusercontent.com/11704566/168319908-0a3683ff-91ee-439a-9121-e50cc276eefc.png)

Thanks

**Comment by irvinm** [comment]
That actually isn't part of the border.  It is another element called the "tab highlighter".

`:root.sidebar { --tab-highlighter-size: 0; }`

**Reply by manyfew** [reply]
Thank you, that's perfect. By the way, where can I find a list of all available elements and their meaning?

**Reply by irvinm** [reply]
Well, to find the variables, I looked into the source-code CSS files.  The challenge is that there is like a base.css and also more specific CSS files for different themes.  I guess you can look over the code as well, or break down the DOM CSS in the debugger.  Both take a little bit of effort.

**Reply by manyfew** [reply]
Thank you @irvinm, so I understand these variables are Firefox built-in variables, not TST specific variables. That at least points me in the right direction.
